OracleGoldenGate/createDB.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
This file should just be reused from the base container and not being duplicated. You can just call `createDB.sh` given that your image is based on the database image.